app/controllers/api/available_slots_controller.rb

```diff
@@ -3,14 +3,15 @@ class AvailableSlotsController < ApiController
     def index
       prison = Prison.find(params.fetch(:prison_id))
 
-      slot_availability = ApiSlotAvailability.new(
-        prison: prison,
-        use_nomis_slots: false
-      )
-      slot_availability.restrict_by_prisoner(
-        prisoner_number: params.fetch(:prisoner_number),
-        prisoner_dob: Date.parse(params.fetch(:prisoner_dob))
-      )
+      slot_availability = ApiSlotAvailability.new(prison: prison,
+                                                  use_nomis_slots: false)
+
+      Timebox.new(TIMEBOX_LIMIT).run do
```

Comment:
There is no fallback here, so what is this doing if it times out?? If the API is still being used then it should return the prison hardcoded availability